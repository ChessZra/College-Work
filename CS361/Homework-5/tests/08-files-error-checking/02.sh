#!/bin/bash
# file does not exist -> 404

PORT=$@

REQUEST=$'GET /tests/doesNotExist/index.html HTTP/1.1\r\n\r\n'

for _ in $(seq 1 3); do
    printf "$REQUEST" | nc 127.0.0.1 $PORT
done
