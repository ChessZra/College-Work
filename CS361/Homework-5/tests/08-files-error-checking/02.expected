HTTP/1.1 404 Not Found

HTTP/1.1 404 Not Found

HTTP/1.1 404 Not Found

